codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeInterceptorSpec.java

```diff
@@ -299,4 +343,8 @@ private TypeName toTypeName(Object valueType) {
         }
         return result;
     }
+
+    private Map<String, ParameterModel> parameters() {
+        return intermediateModel.getEndpointRuleSetModel().getParameters();
```

Comment:
nit: can use `endpointRulesSpecUtils.parameters()` here, or not even have this private method at all.

Reply:
Sounds good, I will remove this method in favor of using the `endpointRulesSpecUtils` defined one.